packages/core/scripts/build-sass.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+for SCSS in src/components/**/*.scss; do
+  OUTPUT=${SCSS%.scss}.module.css
+  HEADER="\
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */\n\
+/* THIS IS GENERATED CODE. Any changes you make will be overwritten! */\n\
```

Comment:
Thank you for this =)

Reply:
It’s the right call! 😄 Glad you mentioned it.